src/checks/hyper-training-31.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  description: 'A maxed IV (31) cannot be hypertrained.',
+  filter (pkmn) {
+    return pkmn.ivHp === 31 || pkmn.ivAtk === 31 || pkmn.ivDef === 31 ||
```

Comment:
I feel like this filter is kind of unnecessary here given that it needs to also be run in the test function anyway. It might be better to remove it.